engine/Cuboid.js

```diff
@@ -0,0 +1,78 @@
+class Cuboid {
+  constructor(width, height, depth, options = {
+    position: [0, 0, 0], rotation: [0, 0, 0]
+  }) {
+    this.width = width;
+    this.height = height;
+    this.depth = depth;
+    this.position = options.position;
+    // this.rotation = options.rotation;
+  }
+
+  at(x, y, z) {
```

Comment:
What is this method for? Translation is already done with the transformation matrix.

Reply:
And as far as I see you can use this method only before converting into a mesh?

Reply:
I want to have a way to position meshes without a GL context, for separating concerns. Like this the mesh itself knows its complete coordinates.  This is just a proposal.

Reply:
I agree with the separating. But I think it has to be done in another way for efficiency reasons. The "ideal" solution would be one which delivers the geometry, which then can be bound to the graphics card and then an array of unique objects which then can be converted into an array of transformations, which then can also be bound to the graphics card. Then we could (theoretically) issue only a single draw call to render every block. 
But I have a proposal how we may improve it, I'll show you on friday :)